tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -144,8 +144,12 @@ try {
       sources.push_back(std::move(source));
    }
 
+   // Always write the RNTuple and the file with the same compression.
+   outFile->SetCompressionSettings(compression);
```

Comment:
Should we "always" do this, or only when there is exactly one RNTuple? What about a file that has one RNTuple (505) and one histogram (101)?

Reply:
I think I agree. At this point, we simply diverged and the default TFile compression is different from the default RNTuple compression. We can discuss if/how we want to address it but I don't think we need extra code in the merger. We have the same situation (different compression algorithms) if you write a new RNTuple.